lite/tests/segmentation/test_dataloader.py

```diff
@@ -0,0 +1,8 @@
+import pytest
```

Comment:
can you add a test where you check that we get all the metrics with 0 values when you pass in a groundtruth and no predictions (see `test_evaluate_segmentation_model_with_no_predictions`)?

Reply:
This isn't possible since the dataclass validator catches a shape mismatch between groundtruths and predictions.

Reply:
See `test_annotation.py::test_segmentation`

Reply:
passing an empty set of predictions was possible in the API, so I see this mismatch causing issues when you go to swap the API's compute functions with `lite`. can you check with Eric to confirm he's cool with no longer allowing empty prediction sets? if yes, then lgtm.

Reply:
thanks for checking!